FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Fhir/SpecificationProviders/Stu3FhirSpecificationProvider.cs

```diff
@@ -0,0 +1,60 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+extern alias FhirStu3;
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Stu3FhirModelInfo = FhirStu3::Hl7.Fhir.Model.ModelInfo;
+
+namespace Microsoft.Health.Fhir.Synapse.Core.Fhir.SpecificationProviders
+{
+    public class Stu3FhirSpecificationProvider : BaseFhirSpecificationProvider
+    {
+        public Stu3FhirSpecificationProvider()
+            : base(null, null)
+        {
+        }
+
+        protected override IEnumerable<string> CompartmentFiles { get; } = null;
+
+        protected override string SearchParameterFile { get; } = null;
+
+        public override IEnumerable<string> GetAllResourceTypes()
+        {
+            return Stu3FhirModelInfo.SupportedResources.Except(ExcludeTypes);
+        }
+
+        public override bool IsValidFhirResourceType(string resourceType)
+        {
+            return Stu3FhirModelInfo.IsKnownResource(resourceType);
+        }
+
+        public override IEnumerable<string> GetCompartmentResourceTypes(string compartmentType)
+        {
+            throw new NotImplementedException();
```

Comment:
GetCompartmentResourceTypes and GetSearchParametersByResourceType can be removed.

Reply:
Sure